client/state/action-types.js

```diff
@@ -273,6 +273,8 @@ export const SELECTED_SITE_SET = 'SELECTED_SITE_SET';
 export const SERIALIZE = 'SERIALIZE';
 export const SERVER_DESERIALIZE = 'SERVER_DESERIALIZE';
 export const SEO_TITLE_SET = 'SEO_TITLE_SET';
+export const SIGNUP_DEPENDENCY_STORE_RESET = 'SIGNUP_DEPENDENCY_STORE_RESET';
+export const SIGNUP_DEPENDENCY_STORE_UPDATE_STATE = 'SIGNUP_DEPENDENCY_STORE_UPDATE_STATE';
```

Comment:
I wasn't quite sure how to name it :) Thanks for the suggestion. Makes more sense to not have `_STATE` in the name.